07_tempConversion/tempConversion.js

```diff
@@ -1,13 +1,17 @@
-const ftoc = function() {
-
+const ftoc = function (far) {
```

Comment:
I would prefer slightly descriptive naming

```suggestion
const ftoc = function (tempF) {
```